### PR TITLE
Feature/40 locationtree

### PR DESCRIPTION
**Pull request by dlin2102**
Basic Location tree. When you click on a leaf/outer most node, it triggers the open drawer.

### CODE REVIEW

**Comment by brian428** [comment]
Dan just FYI for use with the icons, I think this is a list of the possible location_type values:

- HQ (I think there is only one of these, the tree root)
- DIVISION
- DISTRICT
- OPERATING_BASIN (these seem to be named after rivers, but not sure if it means just the actual river, or if it means an area surrounding each river)
- BASIN looks like this is a dam. However there is also a `dam_indicator` field that has different letters in it. I'll try to find out what the letters mean in case it means something important.
- CWMS (not sure about this yet, and this may change to LOCATION)
- STREAM 
- WQ (I think these are water quality sampling locations)

File: src/app-pages/map/components/map-nav-bar/MapNavBar.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+                "Districts",
+                "Projects",
+                "Stream Gages",
+                "Sites",
+                "WQ",
+              ]}
+            />
+          </div>
+          <div className="col-md-4">
+            <div
```
Comment (mohitb3) [inline src/app-pages/map/components/map-nav-bar/MapNavBar.js]: Does the client care about accessibility? I noticed this div isn't reachable via keyboard. If the site has to be 508 compliant then we should make this a button (though the styling would have to be updated as well)
```
            <button
              type="button"
              className="organizational-structure-div"
              onClick={handleOrgDivClick}
            >
              Organizational Structure
            </button>
```

Comment (dlin2102) [reply src/app-pages/map/components/map-nav-bar/MapNavBar.js]: Good point. I'll push up the fix.

**Comment by dlin2102** [comment]
Just fyi, I ended up not using the is_leaf prop bc the plugin generates a hasNodes prop so I decided to stick with that for consistency.

**Pull request merged**